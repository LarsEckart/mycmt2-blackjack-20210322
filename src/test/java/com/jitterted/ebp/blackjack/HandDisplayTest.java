package com.jitterted.ebp.blackjack;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.List;

public class HandDisplayTest {
  @Test
  public void displayFirstCard() throws Exception {
    Hand hand = new Hand(List.of(new Card(Suit.HEARTS, Rank.ACE)));

    Approvals.verify(ConsoleHand.displayFirstCard(hand));
  }

  @Test
  public void displayTwoCard() throws Exception {
    Hand hand =
        new Hand(List.of(new Card(Suit.HEARTS, Rank.ACE), new Card(Suit.SPADES, Rank.THREE)));

    Approvals.verify(ConsoleHand.cardsAsString(hand.cards()));
  }
}
