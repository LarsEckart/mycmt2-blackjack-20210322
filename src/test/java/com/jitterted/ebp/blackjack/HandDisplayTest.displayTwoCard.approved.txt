[31m┌─────────┐[1B[11D│A        │[1B[11D│         │[1B[11D│    ♥    │[1B[11D│         │[1B[11D│        A│[1B[11D└─────────┘[6A[1C[30m┌─────────┐[1B[11D│3        │[1B[11D│         │[1B[11D│    ♠    │[1B[11D│         │[1B[11D│        3│[1B[11D└─────────┘